### PR TITLE
Add contributing guidelines

### LINKED ISSUE #53: Add Contributing.md for new contributors
**Briefly tell us what the proposed feature would do.**
This feature will help new contributors find their way our the CoT website

**Is your feature request related to a problem? If so, please describe the problem.**
No

**Describe the solution you'd like**
A well-formed `contributing.md` file to which we can link users.

**Does this feature replace an older feature? If so, which one?**
No

**Does this feature have an attached Figma(s) file? If so, link here.**
No

**Additional context**

### PR DESCRIPTION
**Pull request by novellac**
**Context**
This PR fixes #53 

**Work done**
- Created `CONTRIBUTING.md` with information about the types of contributions we welcome, and the format in which dictionary contributions should be given.
- Updated `README.md` with more specific installation instructions.

**Testing instructions**
Please read over both files for spelling, grammar, and clarity. Feel free to make commits to the PR!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 
-### 2. Create a Gridsome project
+## Installation
 
-1. `gridsome create my-gridsome-site` to install default starter
-2. `cd my-gridsome-site` to open the folder
-3. `gridsome develop` to start a local dev server at `http://localhost:8080`
-4. Happy coding 🎉🙌
+1. Install Gridsome CLI tool if you don't have it: `npm install --global @gridsome/cli` or `yarn global add @gridsome/cli`
+1. Clone the repo: `git clone https://github.com/cherryontech/website.git`
```
Comment (alliequintano) [inline README.md:13]: do you know why the numbered list shows up as all 1.'s here but in the rendered markdown its the correct list? weird

File: README.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
-# Default starter for Cherry On Tech's Gridsome website
+# Welcome to Cherry On Tech's website
 
-This is the project you get when you run `gridsome create new-project`.
+We are a tech squad! We support each other as we learn and grow in our tech careers. We got the idea from Mo Hampton's JuneteenthConf talk, "Breaking into Tech." Learn more about us in an upcoming blog post!
```
Comment (alliequintano) [inline README.md]: want to add the link to the talk? https://www.youtube.com/watch?v=2N5fI5j-IIs

File: README.md (hunk 1 above)
Comment (novellac) [reply README.md:13]: I'm not sure how exactly it compiles down into the proper numbers, but it makes it easier so when you're writing the markdown file you can rearrange list items.

File: README.md (hunk 2 above)
Comment (novellac) [reply README.md]: Good call, will do.

**Review by alliequintano — APPROVED** [review]

**Pull request merged**